app/Services/Campaign/ExportService.php

```diff
@@ -113,6 +114,46 @@ public function filesize(): int
         return $this->filesize;
     }
 
+
+    protected function campaignModules(): self
+    {
+        $modules = [];
+        $settings = $this->campaign->setting->toArray();
+        unset($settings['id'], $settings['campaign_id'], $settings['created_at'], $settings['updated_at']);
+        $entities = config('entities.ids');
+
+        foreach ($settings as $name => $active) {
+            try {
+                $module = ['enabled' => $active];
+                if (!in_array($name, ['dice_rolls', 'conversations', 'bookmarks', 'inventories', 'entity_attributes', 'assets'])) {
```

Comment:
The whole idea of the try catch was to remove the in array and replace the if with the catch, so that we don't have to remember to update the export code